Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1

```diff
@@ -217,13 +237,14 @@ function Enable-SecureFTPDeploymentForAppServices
     # Setting up context for the current Subscription.
     $context = Set-AzContext -SubscriptionId $SubscriptionId -ErrorAction Stop
     
-    
+    if(-not($AutoRemediation))
+    {
     Write-Host "Subscription Name: [$($context.Subscription.Name)]"
     Write-Host "Subscription ID: [$($context.Subscription.SubscriptionId)]"
     Write-Host "Account Name: [$($context.Account.Id)]"
     Write-Host "Account Type: [$($context.Account.Type)]"
     Write-Host $([Constants]::SingleDashLine)
-    
+    }
```

Comment:
Line 267:

For Autoremediation feature to work, input need to be take from $Path. 
Add that code block

Reply:
Added the code block as discussed.